frontend/src/scenes/web-analytics/WebAnalyticsDashboard.tsx

```diff
@@ -66,20 +68,42 @@ export const Tiles = (props: { tiles?: WebAnalyticsTile[]; compact?: boolean }):
                 compact ? 'gap-x-2 gap-y-2' : 'gap-x-4 gap-y-12'
             )}
         >
-            {tiles.map((tile, i) => {
-                if (tile.kind === 'query') {
-                    return <QueryTileItem key={i} tile={tile} />
-                } else if (tile.kind === 'tabs') {
-                    return <TabsTileItem key={i} tile={tile} />
-                } else if (tile.kind === 'replay') {
-                    return <WebAnalyticsRecordingsTile key={i} tile={tile} />
-                } else if (tile.kind === 'error_tracking') {
-                    return <WebAnalyticsErrorTrackingTile key={i} tile={tile} />
-                } else if (tile.kind === 'section') {
-                    return <SectionTileItem key={i} tile={tile} />
-                }
-                return null
-            })}
+            {featureFlags[FEATURE_FLAGS.WEB_ANALYTICS_EMPTY_ONBOARDING] && !currentTeam?.ingested_event ? (
```

Comment:
Oh, I have forgotten about that property.

I am unsure what the onboarding flow is at the moment, but I remember it would force users to wait for an event to be ingested. Can this be skipped now?